python/test_python.py

```diff
@@ -75,7 +75,7 @@ def test_args(self, docker_container):
 class TestReadJsonFile:
     @pytest.mark.parametrize(
         'docker_container',
-        [["/bin/sh", "-c", "python read_json.py < person-records.json"]],
+        ['python read_json.py < person-records.json'],
```

Comment:
Ha! This only worked because `read_json.py` wasn't reading from stdin, but directly from `person-records.json`. The docker package is being "helpful" by turning `"python read_json.py < person-records.json"` into arguments for us, as if we ran `python 'read_json.py' '<' 'person-records.json'`